### ISSUE TITLE
Console warning related to material UI theme(Typography)

### ISSUE THREAD

**Issue by sunilb0575** [eclipsesource/jsonforms#1274]
**Describe the bug**
Getting console warning related to material UI Theme(Typography)

```
Warning: Failed prop type: Material-UI: you are using a deprecated typography variant: `headline` that will be removed in the next major release
```

**To Reproduce**
1. Go to 'Console'

**Expected behavior**
No warning related to material UI theme

**Suggestion**
Change to 
```
const theme = createMuiTheme({
  typography: {
    useNextVariants: true
  }
})
```
**Screenshots**
<img width="752" alt="screen shot 2019-02-25 at 10 54 52 am" src="https://user-images.githubusercontent.com/30049885/53361008-e57aa180-38eb-11e9-97c8-6d2a7afb38b3.png">

**Browser (please complete the following information):**
 - Browser [e.g. any]

**Used Setup (please complete the following information):**
 - Framework: [e.g. react]
 - RendererSet: [e.g. material]

**Comment by edgarmueller** [comment]
Thanks for the report, we'll look into ti.

**Comment by edgarmueller** [comment]
This should be fixed via #1309
